pkg/builder/group/policy_builder.go

```diff
@@ -136,6 +137,11 @@ func (doNoPlace *DoNotPlaceTogetherPolicyBuilder) WithBuyers(buyers *BuyerPolicy
 	return doNoPlace
 }
 
+func (doNoPlace *DoNotPlaceTogetherPolicyBuilder) OnSellerType(sellerType proto.EntityDTO_EntityType) *DoNotPlaceTogetherPolicyBuilder {
```

Comment:
nitpick, consider changing the method name to OnProvider(...).

Reply:
I have considered, but thought just "Provider" it doesn't reflect the fact that the expected argument is the "type" instead of the "provider" itself.  So I think it's more appropriate to keep the word "Type" there.

Also, I chose "Seller" over "Provider" because the entire policy_builder uses "Seller" a lot.  I thought it will be more consistent to use "Seller" throughout.

Reply:
Chatted with @pallavidn. We're good keeping "OnSellerType".  While it's more preferred to switch to Provider/Consumer on user-facing API methods vs. Buyer/Seller which are rather internal, it will be a change throughout the policy/constraint builders.  We will leave it as another task in the future.